docker-compose.yml

```
@@ -7,6 +7,7 @@ services:
         environment:
             FLASK_APP: app
             FLASK_ENV: production
+            DATA_API_HOST: "https://1312api.tech-bloc-sea.dev"
```

Ideally we would want to be able to easily override this in development without having to edit any git-tracked files. I think something like this might work? Devs could create a `.env` file to override the variable and docker-compose would automatically pick it up:
```suggestion
            DATA_API_HOST: "${DATA_API_HOST:-https://1312api.tech-bloc-sea.dev}"
```

Alternatively, this default could exist as a default for `environ.get` call in `api.py`. What do you think?